src/components/app.tsx

```diff
@@ -37,10 +37,10 @@ class App extends React.Component<Props> {
 
     public componentDidMount = () => {
         const wasWalletConnected = localStorage.getWalletConnected();
-        if (SHOULD_ENABLE_NO_METAMASK_PROMPT && wasWalletConnected) {
-            this.props.onConnectWallet();
-        } else {
+        if (SHOULD_ALWAYS_PROMPT_CONNECT_METAMASK || !wasWalletConnected) {
```

Comment:
Now that I think a little bit more about this:

Since this became super optional and not even listed in the `.env` file, _should we remove it and just leave the `!wasWalletConnected` check_?

Reply:
Yes I think it's better to just leave ` !wasWalletConnected`, makes more sense, at least for me

Reply:
Yeah, I agree too. If @fvictorio or @mariano-aguero add one :+1: more to this comment I'll go ahead and remove it.

Reply:
:+1:

Reply:
Will go ahead as stated on the 1st comment in this thread and mark this as resolved.